float32 min_x
float32 max_x
float32 min_y
float32 max_y
float32 depth
---
geometry_msgs/PointStamped board_center_point

